Terminated
Terminated
Traceback (most recent call last):
  File "train.py", line 357, in <module>
    with open(acc_time_file, 'a') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'out_rmcos/mnist/50/0.9/2022-08-18_14:17:38/acc_time.txt'
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
Terminated
multiple_gpu.sh: line 43: 68333 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 48: 68199 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
Terminated
Terminated
multiple_gpu.sh: line 46: 69400 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 51: 69401 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 47: 69902 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 52: 69903 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 48: 70295 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 53: 70296 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 49: 70606 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 51: 71027 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 54: 70607 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 52: 71416 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 55: 71445 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 53: 71474 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 57: 71524 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 54: 71574 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 58: 71596 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 55: 71625 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 59: 71675 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 57: 71711 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 60: 71761 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 61: 71885 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 58: 71847 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 59: 72127 Aborted                 (core dumped) python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 60: 72503 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 61: poch: command not found
multiple_gpu.sh: line 64: 72015 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.1 --num_labeled 50
multiple_gpu.sh: line 65: num_epoch: command not found
multiple_gpu.sh: line 66: 73053 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 50
multiple_gpu.sh: line 64: 73049 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 50
multiple_gpu.sh: line 67: 73217 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 50
multiple_gpu.sh: line 65: 73218 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 50
multiple_gpu.sh: line 66: 73382 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 50
multiple_gpu.sh: line 69: 73381 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 100
multiple_gpu.sh: line 67: 73507 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 50
multiple_gpu.sh: line 70: 73508 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 100
multiple_gpu.sh: line 69: 73626 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 100
multiple_gpu.sh: line 71: 73627 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 100
multiple_gpu.sh: line 72: 73709 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 100
multiple_gpu.sh: line 70: 73708 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 100
multiple_gpu.sh: line 74: 73865 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 600
multiple_gpu.sh: line 71: 73868 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 100
multiple_gpu.sh: line 75: 73965 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 600
multiple_gpu.sh: line 72: 73966 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 100
multiple_gpu.sh: line 76: 74058 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 600
multiple_gpu.sh: line 74: 74059 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 600
multiple_gpu.sh: line 77: 74204 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 600
multiple_gpu.sh: line 75: 74205 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 600
multiple_gpu.sh: line 76: 74485 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 600
multiple_gpu.sh: line 79: 74484 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 1000
multiple_gpu.sh: line 77: 74591 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 600
multiple_gpu.sh: line 80: 74592 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 1000
multiple_gpu.sh: line 79: 74709 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 1000
multiple_gpu.sh: line 81: 74710 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 1000
multiple_gpu.sh: line 80: 74791 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 1000
multiple_gpu.sh: line 82: 74792 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 1000
multiple_gpu.sh: line 81: 74855 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 1000
multiple_gpu.sh: line 84: 74856 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 3000
multiple_gpu.sh: line 82: 74919 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 1000
multiple_gpu.sh: line 85: 74920 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 3000
multiple_gpu.sh: line 84: 74984 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.1 --num_labeled 3000
multiple_gpu.sh: line 86: 74985 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 3000
multiple_gpu.sh: line 85: 75066 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.3 --num_labeled 3000
multiple_gpu.sh: line 87: 75067 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 3000
multiple_gpu.sh: line 86: 75130 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.5 --num_labeled 3000
multiple_gpu.sh: line 87: 75257 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.7 --num_labeled 3000
multiple_gpu.sh: line 23: 76492 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 2: 75629 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 3: 0: command not found
multiple_gpu.sh: line 24: 77481 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 24: 77477 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 25: 77599 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 25: 77598 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 27: 77677 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 26: 77678 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 50
multiple_gpu.sh: line 28: 77724 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 28: 77725 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 29: 77764 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 29: 77765 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 30: 77806 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 31: 77805 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 32: 77846 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 32: 77847 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 33: 77969 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 33: 77970 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 35: 78016 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 34: 78015 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 36: 78056 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 36: 78055 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 37: 78095 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 37: 78096 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 39: 78136 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 38: 78135 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 40: 78176 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 40: 78175 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 41: 78216 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 41: 78215 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 42: 78255 Killed                  python train.py --num_epoch 50 --mode rlmsoftmax --alpha 0.9 --num_labeled 3000
Traceback (most recent call last):
  File "train.py", line 231, in <module>
    C_scheduler.get_last_lr()[0] = args.lrC
AttributeError: 'ReduceLROnPlateau' object has no attribute 'get_last_lr'
Traceback (most recent call last):
  File "train.py", line 231, in <module>
    C_scheduler.get_last_lr()[0] = args.lrC
AttributeError: 'ReduceLROnPlateau' object has no attribute 'get_last_lr'
Traceback (most recent call last):
  File "train.py", line 231, in <module>
    C_scheduler.get_last_lr()[0] = args.lrC
AttributeError: 'ReduceLROnPlateau' object has no attribute 'get_last_lr'
multiple_gpu.sh: line 6: 78858 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 7: 78987 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 8: 79069 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 10: 79095 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 11: 79117 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 12: 79140 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 14: 79162 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 15: 79184 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 16: 79209 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 18: 79231 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 19: 79257 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 20: 79279 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 387, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 41, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 387, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 41, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
multiple_gpu.sh: line 6: 80507 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 7: 82509 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 8: 82582 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 10: 82615 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 11: 82641 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 12: 82667 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 14: 82710 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 15: 82739 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 16: 82761 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 18: 82788 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 19: 82814 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 20: 82840 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
multiple_gpu.sh: line 7: 84775 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 8: 84955 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 100
multiple_gpu.sh: line 10: 85000 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 11: 85022 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 12: 85044 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 600
multiple_gpu.sh: line 14: 85066 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 15: 85093 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 16: 85120 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 1000
multiple_gpu.sh: line 18: 85147 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 19: 85169 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 20: 85191 Killed                  python train.py --num_epoch 50 --mode rmcos --alpha 0.9 --num_labeled 3000
multiple_gpu.sh: line 10: python: command not found
multiple_gpu.sh: line 11: python: command not found
multiple_gpu.sh: line 12: python: command not found
multiple_gpu.sh: line 14: python: command not found
multiple_gpu.sh: line 15: python: command not found
multiple_gpu.sh: line 16: python: command not found
multiple_gpu.sh: line 18: python: command not found
multiple_gpu.sh: line 19: python: command not found
multiple_gpu.sh: line 20: python: command not found
multiple_gpu.sh: line 10: python: command not found
multiple_gpu.sh: line 11: python: command not found
multiple_gpu.sh: line 12: python: command not found
multiple_gpu.sh: line 14: python: command not found
multiple_gpu.sh: line 15: python: command not found
multiple_gpu.sh: line 16: python: command not found
multiple_gpu.sh: line 18: python: command not found
multiple_gpu.sh: line 19: python: command not found
multiple_gpu.sh: line 20: python: command not found
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 390, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 419, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
multiple_gpu.sh: line 2: python: command not found
multiple_gpu.sh: line 3: python: command not found
multiple_gpu.sh: line 4: python: command not found
multiple_gpu.sh: line 5: python: command not found
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:72: RuntimeWarning: All-NaN slice encountered
  if dtype_str1.startswith("float") and np.nanmin(im) >= 0 and np.nanmax(im) <= 1:
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:93: RuntimeWarning: All-NaN slice encountered
  mi = np.nanmin(im)
/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py:94: RuntimeWarning: All-NaN slice encountered
  ma = np.nanmax(im)
Traceback (most recent call last):
  File "train.py", line 420, in <module>
    visualize_generated_images(epoch, sample_z_, args)
  File "train.py", line 42, in visualize_generated_images
    os.path.join(generated_images_dir, 'epoch%03d' % epoch + '.png'))
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 56, in save_images
    return imsave(images, size, image_path)
  File "/home/asus/Documents/thaodd/rmcosssl/mnist/utils.py", line 61, in imsave
    return imageio.imwrite(path, image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/v2.py", line 196, in imwrite
    return file.write(im, **kwargs)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/legacy_plugin_wrapper.py", line 171, in write
    writer.append_data(image)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/format.py", line 559, in append_data
    return self._append_data(im, total_meta)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/plugins/pillow_legacy.py", line 459, in _append_data
    im = image_as_uint(im, bitdepth=8)
  File "/home/asus/miniconda3/envs/mlenv/lib/python3.7/site-packages/imageio/core/util.py", line 96, in image_as_uint
    raise ValueError("Minimum image value is not finite")
ValueError: Minimum image value is not finite
